pkg/commands/execution_context.go

```diff
@@ -525,6 +528,12 @@ func (ctx *executionContext) completeTask(event operationEvent) {
 	ctx.Lock()
 	delete(ctx.running, event.task.taskID)
 	ctx.completed = append(ctx.completed, event.task)
+	if event.task.test.Status == model.StatusFailed {
+		ctx.failedTestsCount++
+	}
+	if ctx.cloudTestConfig.FailedTestsLimit != 0 && ctx.failedTestsCount == ctx.cloudTestConfig.FailedTestsLimit {
+		ctx.terminationChannel <- errors.Errorf("failed tests limit is reached: %d", ctx.cloudTestConfig.FailedTestsLimit)
```

Comment:
Let's correct text: "Allowed limit for failed tests is reached"

Reply:
Ok, did it